.github/workflows/test-with-pypi.yml

```
@@ -0,0 +1,40 @@
+name: Test with PyPI
+
+on: [pull_request, workflow_dispatch]
+
+env:
+  PYTHONUNBUFFERED: 1
+
+jobs:
+
+  # Test against PyPI packages
+  test-with-pypi:
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        python-version: ['3.11']
```

I think it would be good to extend this to all current Python versions
```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```